routes/api/helData.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const express = require('express');
+const Hel = require("../../models/Hel");
+const router = express.Router();
+
+router.get("/searchHEL/:event", async (req, res) => {
+    try {
+
+        const elective = req.params.event;
+        let courseSlots = "";
+        let result = await Hel.findOne({ courseName: elective });
+        if (result == null) {
+            res.send(null);
+        }
+        else {
+            courseSlots = result.studentsInterestedInSlot;
+            const studentsInterestedInAllSlots = [];
+            for (let slotNo of ['0', '1', '2', '3', '4', '5', '6', '7'])
+                studentsInterestedInAllSlots.push(courseSlots.get(slotNo));
+
+            res.json({ studentsInterestedInAllSlots });
+        }
+    }
+    catch (err) {
+        console.error(err.message);
```

Comment:
Plz Remove this console log!

Reply:
It's part of an error handling routine. We'll need it to display in the console like all other sites